### ISSUE TITLE
[PAD]

### ISSUE THREAD

**Issue by johnbager** [Morizeyao/GPT2-Chinese#218]
为什么我训练出来的输出全是【PAD】
![image](https://user-images.githubusercontent.com/38372764/122918759-0bb48d00-d392-11eb-9708-0b040e610005.png)

**Comment by fourthbrother** [comment]
同问，我也出现了这样的状况，只有自己给的prefix，后面都是pad

**Comment by LemingVan** [comment]
我也遇到了，请问楼上解决了吗？
<img width="1435" alt="屏幕快照 2021-08-11 下午10 59 52" src="https://user-images.githubusercontent.com/46369483/129133353-bd3b1386-25e5-4f6c-87fe-01c66dcb10a0.png">

**Comment by liangcaihua** [comment]
outputs = model.forward(input_ids=batch.input_ids, labels=batch.input_ids, attention_mask=batch.attention_mask)
  attention_mask参数设置了没?
